core/src/main/java/ai/timefold/solver/core/api/score/stream/common/ConcurrentUsageInfo.java

```diff
@@ -0,0 +1,16 @@
+package ai.timefold.solver.core.api.score.stream.common;
+
+public interface ConcurrentUsageInfo<Interval_, Point_ extends Comparable<Point_>, Difference_ extends Comparable<Difference_>> {
+
+    /**
+     * @return never null, an iterable that iterates through the interval clusters
+     *         contained in the collection in ascending order
+     */
+    Iterable<ConcurrentUsage<Interval_, Point_, Difference_>> getConcurrentUsages();
+
+    /**
+     * @return never null, an iterable that iterates through the breaks contained in
+     *         the collection in ascending order
+     */
+    Iterable<Break<Point_, Difference_>> getBreaks();
```

Comment:
Just out of curiosity... can you think of a term for a "break" inbetween two ranges of numbers? I'm thinking maybe maths has a term for this.

Reply:
We probably do not want to go to full-on `math` for this; math terms can be non-intuitive. What seems to be common (although without a definition defining it formally) is "gap": https://oeis.org/search?q=gap&language=english&go=Search)

However, it should be consistent with the consecutive collector, which uses "breaks".

Reply:
On the contrary, I'd actually argue that the two collectors don't need to have anything in common; they are different collectors for different use cases, they don't need to share naming. Since we're going for `connected ranges`, I think `gaps` makes sense in this context. 

I have "break" more associated with a gap between two periods of time, which is needlessly specific for this case, I think.